src/snowflake/snowpark/dataframe_reader.py

```diff
@@ -1173,6 +1177,11 @@ def dbapi(
                 For example, `"SET TRANSACTION ISOLATION LEVEL READ UNCOMMITTED"` can be used in SQL Server
                 to avoid row locks and improve read performance.
                 The `session_init_statement` is executed only once at the beginning of each partition read.
+            imports: Name of stage files to import for udtf ingestion.
+            packages: The name and version number of packages required as dependencies.
+            external_access_integration: A string of name of external access integration, if is not None, a server
+                ingestion using UDTF will be used.
+            use_udtf_ingestion: A bool value decide whether to use udtf to ingest external data source, default is False.
```

Comment:
we'd better document what kind of benefits does it provide to users compared to the parquet approach